azuredeploy.json

```diff
@@ -0,0 +1,110 @@
+{
+    "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "siteName": {
+            "type": "string"
+        },
+        "luisAuthoringKey": {
+            "type": "string"
+        },
+        "siteLocation": {
+            "type": "string",
+            "defaultValue": "West US"
+        },
+        "conversationLearnerDockerContainer": {
+            "type": "string",
+            "defaultValue": "DOCKER|conversationlearner/cl-sdk-sample:latest"
+        },
+        "alwaysOn": {
+            "type": "bool",
+            "defaultValue": false
+        }
+    },
+    "variables": {
+        "hostingPlanName": "[parameters('siteName')]"
+    },
+    "resources": [
+        {
+            "apiVersion": "2018-02-01",
+            "name": "[parameters('siteName')]",
+            "type": "Microsoft.Web/sites",
+            "location": "[parameters('siteLocation')]",
+            "kind": "app,linux,container",
+            "tags": {},
+            "dependsOn": [
+                "[resourceId('Microsoft.Web/serverfarms', variables('hostingPlanName'))]"
+            ],
+            "properties": {
+                "name": "[parameters('siteName')]",
+                "siteConfig": {
+                    "appSettings": [
+                        {
+                            "name": "LUIS_AUTHORING_KEY",
+                            "value": "[parameters('luisAuthoringKey')]"
+                        },
+                        {
+                            "name": "NODE_ENV",
+                            "value": "development"
+                        },
+                        {
+                            "name": "WEBSITES_PORT",
+                            "value": "3978"
+                        }
+                    ]
+                },
+                "appCommandLine": "",
+                "alwaysOn": "[parameters('alwaysOn')]",
```

Comment:
Should `alwaysOn` not be in the `siteConfig` object? Currently this ARM template throws error on deploying related to the `alwaysOn` property.

Reply:
Thanks for pointing this out. `azuredeploy.net` didn't show any issues with the arm template. The following change 21fbe683eff39c67d3e7fdad70c866856e5ae8f6 should address this issue.